packages/storage-mongo/src/lib/event.js

```diff
@@ -16,9 +16,13 @@ function createEventModel({ mongoose }) {
       parentId: { type: String, default: null },
       title: { type: String },
       status: { type: String, enum: ["open", "done"], default: "open" },
-      labels: { type: [String] },
+      // DO NOT CHANGE THE labels DEFAULT VALUE UNLESS YOU KNOW WHAT YOU'RE DOING
+      // https://stackoverflow.com/a/20714461/526426
+      labels: { type: [String], default: undefined },
```

Comment:
If the default is undefined, will it use this when we not send the field at all?

Reply:
yes. we need the undefined default for the "reload browser" case. this prevents mongoose from sending empty arrays all over the place.